charts/metallb/templates/speaker/psp.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if and .Values.psp.create .Values.speaker.psp.create -}}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
```

Comment:
Something to keep in mind for the future (we likely want to open a ticket to track this, not only for MetalLB but for all charts we embed): PodSecurityPolicy is deprecated starting in K8S 1.21, and will be removed in 1.25. See https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/

Reply:
Opened #3437